Automattic/wp-calypso
# Signup: Reduced google error message

Context: https://github.com/Automattic/wp-calypso/issues/86884#issuecomment-1911695886

## Problem

![image](https://github.com/Automattic/wp-calypso/assets/52076348/28c30cbd-574f-44b0-82ed-c62a78baf412)

In signup, the error message looks a bit awkward for the social first screen (copy too long, container too small)

## Solution

Switched to: 

![image](https://github.com/Automattic/wp-calypso/assets/52076348/a83923e6-965b-4c55-99bf-1ebca4227dc7)

## Testing

1. Create a WP account with a Google email or make sure Google login is not connected. https://wordpress.com/me/security/social-login
2. Log out
3. Go to https://wordpress.com/start/
4. Continue to the account creation step
5. Sign up via Google
6. Check the error message.
